multinet/db.py

```diff
@@ -58,42 +58,41 @@
     port=int(os.environ.get("ARANGO_PORT", "8529")),
     protocol=os.environ.get("ARANGO_PROTOCOL", "http"),
 )
-restricted_keys = {"_rev", "_id"}
 
 
-def db(name: str) -> StandardDatabase:
+def db(name: str, readonly: bool = True, immediate: bool = False) -> StandardDatabase:
```

Comment:
What does `immediate` do?

Reply:
Initializes the connection to the database immediately by sending a test request. This is generally not desired, as it means more upfront requests when we may not need them.

Reply:
Why did you include it? Do we require immediate access somewhere?

Reply:
Not sure, I can remove it.